config/config.go

```diff
@@ -168,6 +169,7 @@ func (c *Config) AirbrakeTlsConfig() (*tls.Config, error) {
 			return nil, fmt.Errorf("can't properly load cert pair (%s, %s): %s", certPath, keyPath, err.Error())
 		}
 		tlsConfig.Certificates = []tls.Certificate{cert}
+		// nolint:staticcheck
```

Comment:
This one should probably have a TODO to fix properly eventually.

Reply:
added a TODO. Will file an issue to fix it in a later PR

Reply:
:) Technically it should be "remove the lint `bypass`" because we want to add the check back, not remove it again..